packages/block-library/src/gallery/index.php

```diff
@@ -130,7 +116,58 @@ function block_core_gallery_render( $attributes, $content ) {
 			'context' => 'block-supports',
 		)
 	);
-	return (string) $processed_content;
+
+	// Randomize the order of image blocks. Ideally we should shuffle
+	// the `$parsed_block['innerBlocks']` via the `render_block_data` hook.
+	// However, this hook doesn't apply inner block updates when blocks are
+	// nested.
+	// TODO: In the future, if this hook supports updating innerBlocks in
+	// nested blocks, it should be refactored.
+	// See: https://github.com/WordPress/gutenberg/pull/58733
```

Comment:
```suggestion
	// See: https://github.com/WordPress/gutenberg/pull/58733.
```

I get a linting error for the inline comment here not ending in a `.` Is there an official recommendation for adding these reference links. Does it hurt to add the period to the end of it to reduce noise from the linter?

Reply:
Looking at the entire core and Gutenberg code, it seems like using the `@see` annotation is a good idea. Updated with [5ce2ec0](https://github.com/WordPress/gutenberg/pull/58733/commits/5ce2ec01ef6a74ddaff12a781dd18c7f3e68ebdc).